tests/integration/query.ts

```diff
@@ -50,3 +58,130 @@ it('universal autocomplete sets search intents', async () => {
   expect(statefulCore.state.query.searchIntents).toEqual(['NEAR_ME']);
 });
 
+it('vertical autocomplete get latest request results', async () => {
+  const mockedCore: any = {
+    verticalAutocomplete: jest.fn(
+      async (_request: VerticalAutocompleteRequest) => {
+        const waitTime = _request.input?.length * 200;
```

Comment:
clever haha

Reply:
honestly we probably don't need to wait 200 seconds, I did some similar race conditions tests (that we ended up scraping) here -> https://github.com/yext/answers-headless/pull/25/commits/650eaedfeafcf2ed2f0613440a6524aedd09b612
and only waited 0, 1, and 2 ms.